ccd-definition/AuthorisationCaseField.json

```diff
@@ -153,6 +153,20 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicants",
+    "UserRole": "caseworker-publiclaw-solicitor",
```

Comment:
Other roles except solicitor and system update user should also have access to new field otherwise migrated structure won't appear in people tab for HMCTS admin, CAFCASS and gate keeper.

Reply:
👍